.github/workflows/ci-go.yml

```diff
@@ -40,6 +40,11 @@ jobs:
           go-version-file: 'go.work'
           check-latest: true
 
+      - name: 'Install Bingo'
+        run: |
+          go install github.com/bwplotka/bingo@latest
+          bingo get --link github.com/bwplotka/bingo
```

Comment:
We shouldn't need this change, here's why:

Bingo is basically just fancy `go vendor` but for tools. This means that we shouldn't need to use `bingo get` directly in CI, because the `.bingo` directory should already be built and committed as code in the repo. We'll be able to use any installed tool via the included `.mk` in any subsequent `make` calls, or, we can  `. .bingo/variables.env` file bingo already built for us for `/bin/sh`. After those files get sourced for their respective `make` or `sh` environments, we can use the variables they define (in our case, we'll have `${MOCKGEN}` that is a path to the "vendored" binary.

Reply:
How does the Bingo executable otherwise get installed in a CI pipeline?

The first line just installs `$(GOBIN)/bingo`.
The second line establishes a versioned softlink `$(GOBIN)/bingo-v0.9.0` that our [.bingo/Variables.mk wants](https://github.com/Azure/ARO-HCP/blob/35408ec1b539bbfd69eb29c56017fb81801e4079/.bingo/Variables.mk#L20).

So this is just to bootstrap Bingo itself, not install other tools under Bingo's control.

Reply:
Force-pushed an update that removes this hunk.